storage/innobase/log/log0log.cc

```diff
@@ -62,7 +62,7 @@ Created 12/9/1995 Heikki Tuuri
 
 #include "system_key.h"
 
-static redo_log_encrypt_enum found_log_encryption_mode = REDO_LOG_ENCRYPT_OFF;
+redo_log_encrypt_enum existing_redo_encryption_mode = REDO_LOG_ENCRYPT_OFF;
```

Comment:
IMHO "current" is a better fit than "existing", but this is up to you

Reply:
This could be true even if the actual setting is off (if the log has previously encrypted records) - I think current would be misleading.